### ISSUE TITLE
Fallo cargar api

### ISSUE THREAD

**Issue by JoseAngelAl** [JoseAngelAl/practica_dwec_gestor_presupuesto#9]
Hola @pedroprieto, tengo un problema al cargar la aplicación, me tira un error al usar la función repintar y no se porqué.

![image](https://user-images.githubusercontent.com/91339158/147777522-82fab898-10ac-4f72-bea4-c37cf851ae24.png)

le estoy dando vueltas y no consigo verlo

**Comment by pedroprieto** [comment]
Buenas, @JoseAngelAl 

¿Puedes indicarme la URL de la API que estás utilizando? Con tu nombre de usuario.

Un saludo!

**Comment by JoseAngelAl** [comment]
https://suhhtqjccd.execute-api.eu-west-1.amazonaws.com/latest/joseangelalfonso @pedroprieto

**Comment by pedroprieto** [comment]
Buenas,
El problema es que has creado un gasto de manera incorrecta: tiene un array dentro de un array
![imagen](https://user-images.githubusercontent.com/7632447/148912323-d6548169-8b8e-4716-8ee4-45695b412e83.png)

Además, hay otro que no tiene el campo `etiquetas` creado. Por ello, el cliente produce un fallo.

He actualizado los datos borrando a través de https://hoppscotch.io/es los gastos incorrectos. Prueba de nuevo a ver si funciona.

Un saludo!!

**Comment by JoseAngelAl** [comment]
correcto @pedroprieto ya me funciona, era eso, esto hay que cogerlo con pinzas jeje

**Comment by pedroprieto** [comment]
Sí, es que la implementación de la API es muy básica :blush: Básicamente se traga todo lo que le eches :rofl:
